content/en/synthetics/browser_tests/_index.md

```diff
@@ -226,6 +226,82 @@ You can switch tabs in a browser test recording in order to perform an action on
 6. Once you have finished your scenario, click **Save and Launch Test**.
 
+## Step Replay
+
+Step Replay lets you re-run one or more steps of your browser test directly from the Datadog Synthetics Chrome extension. This feature is designed to help you quickly set up the right state when adding or editing steps in the middle of a long test, without having to manually replay the entire journey from scratch.
+
+<div class="alert alert-warning">
+  <p>The current version of the extension<strong> <span style="font-weight: bold;">does not have Chrome’s debugger permission</span></strong>, as a result:</p>
+  <ul>
+    <li>Some advanced actions are <strong>not yet supported</strong>.</li>
+    <li>Certain steps (like <strong>click</strong> or <strong>hover</strong>) may behave <strong>differently</strong> than in a full Synthetics test run.</li>
+    <li><strong>JavaScript-based assertions or extractions, keystroke simulations, and email interactions</strong> are <strong>currently unavailable</strong>.</li>
+  </ul>
+</div>
+
+### How to use Step Replay
+
+You can replay steps in three ways:
+
+<strong>1. Single Step Replay:</strong> Re-execute a single step:
+{{< img src="synthetics/browser_tests/recording__replay--replay-one-step.mp4" alt="Single Step Replay" video="true" height="400px" >}}
+<p style="text-align: center;"><em>Hover the step, click on the play button to replay only this step</em></p>
+
+<strong>2. Replay All Steps:</strong> Run the entire sequence of steps as currently defined in the recorder:
+{{< img src="synthetics/browser_tests/recording__replay--replay-all-steps.mp4" alt="Replay All Steps" video="true" height="400px" >}}
+<p style="text-align: center;"><em>Click on the replay all icon (⏩︎) on top of the step list to replay all steps</em></p>
+
+<strong>3. Replay Selected Steps:</strong> Run a subset of steps you select in the step list:
+{{< img src="synthetics/browser_tests/recording__replay--replay-selected-steps.mp4" alt="Replay Selected Steps" video="true">}}
+<p style="text-align: center;"><em>Select the steps you want to replay then click on the replay selected icon (⏩︎) on top of the step list.</em></p>
+
+### Fully Supported Steps
+
+These steps are fully supported in the current version of Step Replay, and behave consistently with the Synthetics platform:
+
+- Extract Variable
+- Go to URL
+- Refresh
+- Scroll
+- Select Option
+- Wait
+- Assert Checkbox State
+- Assert Current URL
+- Assert Element Attribute
+- Assert Element Content
+- Assert Element Present
+- Assert File Download
+- Assert Page Contains
+- Assert Page Lacks
+
+### Debugger Permissions & Upcoming Improvements
+
+Currently, the Step Replay feature runs without Chrome’s debugger permission. This means some advanced user actions are either not yet supported or may behave differently compared to full Synthetics test runs.
```

Comment:
```suggestion
Currently, the step replay feature runs without Chrome's debugger permission. This means some advanced user actions are either not yet supported or may behave differently compared to full Synthetic Monitoring test runs.
```
This implies some future change in functionality but I think this is ok to state.